historyserver/Makefile

```diff
@@ -0,0 +1,140 @@
+# Image URL to use all building/pushing image targets
+GOLANGCILINT_VERSION ?= v1.59.0
+GOBIN := $(shell go env GOPATH)/bin
+GOBIN_GOLANGCILINT := $(GOBIN)/golangci-lint
+
+DOCKERBUILDER_INSTANCE=historyserver
+OUT_DIR=output
+BIN_DIR=$(OUT_DIR)/bin
+BINARY_NAME=historyserver
+BINARY_NAME_COLLECTOR=collector
+
+# Setting SHELL to bash allows bash commands to be executed by recipes.
+# Options are set to exit when a recipe line exits non-zero or a piped command fails.
+SHELL = /usr/bin/env bash -o pipefail
+BUILD_TIMESTAMP = $(shell date -u +"%Y-%m-%dT%H:%M:%SZ")
+COMMIT_SHORT ?= $(shell git rev-parse --short HEAD)
+BRANCH ?= $(shell git branch --show-current)
+VERSION ?= $(shell git describe --tags --long|awk -F '-' '{print $$1"."$$2"-"$$3""}')
+
+PACKAGE = gitlab.alibaba-inc.com/eml/historyserver
+
+GO_LDFLAGS := -extldflags "-static"
+# GO_LDFLAGS += -w -s # Drop debugging symbols.
+GO_LDFLAGS += -X $(PACKAGE)/pkg.Version=$(VERSION) \
+	-X $(PACKAGE)/pkg.CommitID=$(COMMIT_SHORT) \
+	-X $(PACKAGE)/pkg.BuildDate=$(BUILD_TIMESTAMP) \
+	-X $(PACKAGE)/pkg.Branch=$(BRANCH)
+GO_BUILD_FLAGS := -ldflags '$(GO_LDFLAGS)'
+
+GOOS ?= darwin
+GOARCH ?= amd64
+
+.PHONY: all
+
+all: build
+
+.PHONY: clean
+clean:
+	rm -rf $(OUT_DIR)
+
+.PHONY: build
+build: buildcollector buildhistoryserver
```

Comment:
should the buildhistoryserver should be removed because it is not ready in this pr? And so does the historyserver related content in this file.